playbook-website/config/webpack/environment.js

```diff
@@ -2,28 +2,56 @@ const path = require('path')
 
 const { environment } = require('@rails/webpacker')
 
-environment.loaders.insert('javascript', {
-  test: /\.(js|jsx)$/,
+const babelLoader = environment.loaders.get('babel')
+babelLoader.test = /\.(js|jsx|ts|tsx)?(\.erb)?$/
```

Comment:
`babel-loader` in Webpacker includes `mjs` (ES modules) in it's test expression by default. It is a known issue that, in modern systems, this can fail with errors like:

```zsh
ERROR in /Users/xxx/code/playbook-icons/packages/playbook-icons-react/dist/esm/Accessories.mjs 17:12-31
Can't import the named export 'createElement' from non EcmaScript module
```

Which means ES modules aren't configured as the correct loader `type`